OJP/OJP_RequestSupport.xsd

```diff
@@ -161,11 +161,33 @@
 			<xs:element name="Place" type="PlaceStructure" maxOccurs="unbounded"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="OperatorsStructure">
+		<xs:annotation>
+			<xs:documentation>Structure providing a collection of operators.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="OperatorsStructure" type="siri:AffectedOperatorStructure" maxOccurs="unbounded">
+				<xs:annotation>
+					<xs:documentation>Operators of the services.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="OperatorsRefStructure">
+		<xs:sequence>
+			<xs:element name="OperatorRef" type="siri:OperatorRefStructure" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Why unbounded?

Reply:
to have a list of refs (e.g. all taxi operators in Zürich)

Reply:
In `OJP_JourneySupport.xsd` you define a `OperatorRefList` consisting of the same definition as the `OperatorsRefStructure` defined here in `OJP_RequestSupport.xsd`. The definition of `OperatorsRefStructure` here is not even used in `OJP_Request` but `OJP_Places.xsd`. This definition should probably done in `OJP_Common.xsd` instead?

Reply:
simplified